src/gameplay.ts

```diff
@@ -3,14 +3,17 @@ import Card from './card';
 export default class Gameplay {
     public playerMoves: number = 0;
     public cards: Card[] = [];
+    public sameCards: number;
+    public timeToCompleteGame: number;
 
-    constructor(cards: Card[]) {
+    constructor(cards: Card[], sameCards: number) {
         this.cards = cards;
+        this.sameCards = sameCards;
     }
 
     public showCard(clickedCardId: number): void {
         this.playerMoves++;
-        if (this.cards[clickedCardId].pairFound == false) {
+        if (this.cards[clickedCardId] && this.cards[clickedCardId].pairFound == false) {
```

Comment:
nie lepiej === ?

Reply:
https://www.java67.com/2013/07/difference-between-equality-strict-vs-operator-in-JavaScript-Interview-Question.html

Reply:
dotyczy to całego kodu gdzie możw tak występować